content/help/ops/security/health-checks-and-mtls/index.md

```diff
@@ -3,7 +3,6 @@ title: Health Checks and Mutual TLS
 description: How to get health checks working when mutual TLS is enabled.
 weight: 40
 ---
-As of Istio 1.0, we support ‘permissive’ mode so that your service can take both http and mutual TLS traffic. The
-caveat is that you can’t [TBD]
-
-Refer the health check page for more info.
+As of Istio 1.0, we support a ‘permissive’ mode so that your service can take both mTLS and plaintext traffic.
```

Comment:
To use the proper markup and capitalization when doing code references, I suggest:
As of Istio 1.0, you can enable a `PERMISSIVE` mode for your service to take both mTLS and plain-text traffic.